### PR TITLE
Install docs

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes https://github.com/Unidata/cftime/issues/31

### CODE REVIEW

File: docs/installing.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Installation
+============
+
+Required dependencies
+---------------------
+
+- Python 2.7, 3.4, 3.5, or 3.6
+- `numpy <http://www.numpy.org/>`__ (1.7 or later)
+- `cython <http://cython.org/>`__ (0.19 or later)
```
Comment (shoyer) [inline docs/installing.rst]: I think Cython should only required for the development version? That was the case for netCDF4.

Comment (ocefpaf) [reply docs/installing.rst]: You are correct. I'll fix that in a moment.
(Old habit, from before wheels, when people needed the build tool to pip install a package.)

**Comment by jhamman** [comment]
@ocefpaf - the build is failing because the `installing.rst` document isn't in a TOC tree yet. Can you add it to the index page? Otherwise, I think this looks okay.

**Comment by ocefpaf** [comment]
> @ocefpaf - the build is failing because the installing.rst document isn't in a TOC tree yet. Can you add it to the index page? Otherwise, I think this looks okay.

Sure. I [asked where it should be](https://github.com/Unidata/cftime/pull/47#discussion_r187974986) :smile:

Is it OK to be right after `Indices and tables`?

File: docs/installing.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
+Installation
```
[the remark this answers is not in the document]
Comment (jhamman) [reply docs/installing.rst]: I think this should go above the API. So maybe 

```rst
cftime
======

Python library for decoding time units and variable values in a netCDF file conforming to the Climate and Forecasting (CF) netCDF conventions.

Installing
----------

...quick line pointing people to installation inspection

API
---
...

Tables and ...
--------------
```

**Comment by jhamman** [comment]
@ocefpaf - I took the liberty to make the headings a bit smaller but other than that, this looks good. Thanks for the help here. Provided the tests pass, I'll merge this and create a release tag!

**Comment by ocefpaf** [comment]
> @ocefpaf - I took the liberty to make the headings a bit smaller

Whatever works :smile: 

> but other than that, this looks good. Thanks for the help here. Provided the tests pass, I'll merge this and create a release tag!

Awesome! Thanks!!

**Pull request merged**